app/lib/provider/persistence_provider.dart

```diff
@@ -233,6 +238,14 @@ class PersistenceService {
     await _prefs.setInt(_portKey, port);
   }
 
+  int getAnonymousAccess() {
+    return _prefs.getInt(_anonymousAccess) ?? 0;
```

Comment:
If you intend to have 3 or more options, then an enum should be used (see `getSendMode`, `setSendMode` below).
An `int` is not type safe enough which could lead to bugs.

Reply:
> Thinking again about this: You commented somewhere that `2` is only temporary. Then we only need to persist 2 states. In this case, a `bool` should be used here.

if use 2 states, only on/off can be use  ; 
in the share via link page, there is also a switch for temporay enablement, and it can't affect the global configuration.
so needs 3 states for this functional， globalOn/ globalOff/ onceOn
i will refactor code to enums later,

Reply:
or use 2 configuration to done this , one use to global config, one use to temporary config

but i think int is better use to,  first Bit use to global config, and seceond Bit use to  temporary config

Reply:
> Thinking again about this: You commented somewhere that `2` is only temporary. Then we only need to persist 2 states. In this case, a `bool` should be used here.

i already refactor to bool